.github/workflows/tpu_tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Keras TPU Tests
+
+# TODO: Consider enabling all tests (pytest, applications, etc.) with NNX in the future
+# Currently only basic flow tests run with NNX enabled
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```

Comment:
Remove `pull_request` here. They're always excluded by the `if` below so it just creates a "skipped" entry.